CITATION.cff

```diff
@@ -0,0 +1,181 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: MESSAGEix
+message: >-
+  Thank you for using the MESSAGEix framework! Please cite
+  it using the metadata from this file :)
```

Comment:
Since you put the original 2019 MESSAGEix article in the references, should that one be cited as well or only the DOI of this repo?

Reply:
Yes, please use a statement like "If you use the MESSAGEix framework, please refer to the GitHub repository and cite the MESSAGEix framework manuscript (https://doi.org/10.1016/j.envsoft.2018.11.012)"